### PR TITLE
Add ci test

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

Moving the chart file into a /charts dir for test.

### CODE REVIEW

**Comment by yeya24** [comment]
I don' t know why there is no circle ci triggered for this PR?

**Comment by allencloud** [comment]
I see no follow on this project:
屏幕快照 2019-12-01 下午7 20 44

**Comment by yeya24** [comment]
image
I set up this project but it did not work. Maybe could you please help set up it in your circle ci?

File: test/kind-config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+  
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
```
Comment (starnop) [inline test/kind-config.yml]: Please type an enter at the end of the file.

Comment (yeya24) [reply test/kind-config.yml]: Done

**Pull request closed without merging**